.circleci/cimodel/data/simple/util/docker_constants.py

```diff
@@ -0,0 +1,41 @@
+AWS_DOCKER_HOST = "308535385114.dkr.ecr.us-east-1.amazonaws.com"
+
+DOCKER_IMAGE_BASIC = "/".join([
+    AWS_DOCKER_HOST,
+    "pytorch",
+    "pytorch-linux-xenial-py3.6-gcc5.4:9a3986fa-7ce7-4a36-a001-3c9bef9892e2",
+])
+
+DOCKER_IMAGE_CUDA_10_2 = "/".join([
+    AWS_DOCKER_HOST,
+    "pytorch",
+    "pytorch-linux-xenial-cuda10.2-cudnn7-py3-gcc7:9a3986fa-7ce7-4a36-a001-3c9bef9892e2",
+])
+
+DOCKER_IMAGE_BAZEL = "/".join([
+    AWS_DOCKER_HOST,
+    "pytorch",
+    "pytorch-linux-xenial-py3.6-gcc7:f990c76a-a798-42bb-852f-5be5006f8026",
+])
+
+
+def gen_mobile_docker_name(specifier):
+
+    final_path_part = "".join([
+        "pytorch-linux-xenial-py3-clang5-",
+        specifier,
+        ":9a3986fa-7ce7-4a36-a001-3c9bef9892e2",
+    ])
```

Comment:
Let's introduce a constant denoting build tag, to highlight that all docker images should use the same.

For example, bazel builds should not be using special image, just the same one as `pytorch-linux-xenial-py3.6-gcc7-buid` job is using, 
```suggestion
DOCKER_BUILD_TAG = ":9a3986fa-7ce7-4a36-a001-3c9bef9892e2"
DOCKER_IMAGE_BASIC = "/".join([
    AWS_DOCKER_HOST,
    "pytorch",
    "pytorch-linux-xenial-py3.6-gcc5.4" + DOCKER_BUILD_TAG,
])

DOCKER_IMAGE_CUDA_10_2 = "/".join([
    AWS_DOCKER_HOST,
    "pytorch",
    "pytorch-linux-xenial-cuda10.2-cudnn7-py3-gcc7" + DOCKER_BUILD_TAG,
])

DOCKER_IMAGE_BAZEL = "/".join([
    AWS_DOCKER_HOST,
    "pytorch",
    "pytorch-linux-xenial-py3.6-gcc7" + DOCKER_BUILD_TAG,
])


def gen_mobile_docker_name(specifier):

    final_path_part = "".join([
        "pytorch-linux-xenial-py3-clang5-",
        specifier,
        DOCKER_BUILD_TAG,
    ])
```